identity-android/src/main/java/com/android/identity/android/direct_access/DirectAccessDocumentMetadata.kt

```diff
@@ -13,6 +13,8 @@ interface DirectAccessDocumentMetadata: DocumentMetadata {
     var directAccessDocumentSlot: Int
 
     override suspend fun documentDeleted() {
-        DirectAccess.clearDocumentSlot(directAccessDocumentSlot)
+        if (directAccessDocumentSlot >= 0) {
```

Comment:
I have a similar fix in https://github.com/openwallet-foundation-labs/identity-credential/pull/880. +1 to this.